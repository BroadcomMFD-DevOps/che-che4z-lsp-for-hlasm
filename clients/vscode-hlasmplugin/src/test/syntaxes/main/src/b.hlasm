.L20     MNOTE 4,'Length of block does not exceed 256 (&LBL DSECT), &&CX12345678
               CLEANUP option can be optimized to YES,OP2              X12345678
               blabla,jdios' dfsd fsd sdf s

.L20     MNOTE 4,'Length of block does not exceed 256 (&LBL DSECT), &&CX12345678
               LEANUP option can be optimized to YES,OP2                12345678
    blabla,jdios'

.L20     MNOTE 4,'Length o   lock does not exceed 256 (&LBL DSECT), &&CX12345678
               LEANUP option can be optimized to YES' OP2               12345678

.L20     MNOTE 4,'Length of block does not exceed 256 (&LBL DSECT), &&CX12345678
               LEANUP option can be optimized to YES',OP2OP2OP2OP2OP2OA-12345678
               REMARK,REMARK

assa

LABEL INSTR OP1,OP2,   REMARKS                                         -12345678
               OP3,OP4,       1                                        -12346578
               OP5,OP6                                                 -12346578
               OP6            1                                        -12346578
               OP7                                                      12345678
               
.L20     MNOTE 4,'Length of block does not exceed 256 (&LBL DSECT), &&CX12345678
               LEANUP option can be optimized to YES ,OP2               12345678

LABEL INSTR OP1,OP2    REMARKS                                         -12345678
               OP3 OP4,       1                                        -12346578
               OP5,OP6                                                 -12346578
               OP6            1                                        -12346578
               OP7                                                      12345678
               