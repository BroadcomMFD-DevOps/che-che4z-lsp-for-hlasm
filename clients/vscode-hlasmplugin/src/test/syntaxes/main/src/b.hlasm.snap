>.L20     MNOTE 4,'Length of block does not exceed 256 (&LBL DSECT), &&CX12345678
#^^^^ source.hlasm keyword.control.hlasm
#    ^^^^^^^^^^ source.hlasm entity.name.function.hlasm
#              ^ source.hlasm
#               ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                ^ source.hlasm
#                 ^ source.hlasm string.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                                       ^^^^ source.hlasm variable.other.readwrite.hlasm
#                                                           ^^^^^^^^^ source.hlasm string.hlasm
#                                                                    ^^ source.hlasm string.hlasm
#                                                                      ^ source.hlasm string.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               CLEANUP option can be optimized to YES,OP2              X12345678
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               blabla,jdios' dfsd fsd sdf s
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                            ^ source.hlasm
#                             ^^^^ source.hlasm comment.hlasm
#                                 ^ source.hlasm
#                                  ^^^ source.hlasm comment.hlasm
#                                     ^ source.hlasm
#                                      ^^^ source.hlasm comment.hlasm
#                                         ^ source.hlasm
#                                          ^ source.hlasm comment.hlasm
>
>.L20     MNOTE 4,'Length of block does not exceed 256 (&LBL DSECT), &&CX12345678
#^^^^ source.hlasm keyword.control.hlasm
#    ^^^^^^^^^^ source.hlasm entity.name.function.hlasm
#              ^ source.hlasm
#               ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                ^ source.hlasm
#                 ^ source.hlasm string.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                                       ^^^^ source.hlasm variable.other.readwrite.hlasm
#                                                           ^^^^^^^^^ source.hlasm string.hlasm
#                                                                    ^^ source.hlasm string.hlasm
#                                                                      ^ source.hlasm string.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               LEANUP option can be optimized to YES,OP2                12345678
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                                                       ^^^^^^^^^ source.hlasm invalid.ignored
>    blabla,jdios'
#^^^^^^^^^^^^^^^^^ source.hlasm entity.name.function.hlasm
>
>.L20     MNOTE 4,'Length o   lock does not exceed 256 (&LBL DSECT), &&CX12345678
#^^^^ source.hlasm keyword.control.hlasm
#    ^^^^^^^^^^ source.hlasm entity.name.function.hlasm
#              ^ source.hlasm
#               ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                ^ source.hlasm
#                 ^ source.hlasm string.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                                       ^^^^ source.hlasm variable.other.readwrite.hlasm
#                                                           ^^^^^^^^^ source.hlasm string.hlasm
#                                                                    ^^ source.hlasm string.hlasm
#                                                                      ^ source.hlasm string.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               LEANUP option can be optimized to YES' OP2               12345678
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                                     ^ source.hlasm
#                                                      ^^^ source.hlasm comment.hlasm
#                                                         ^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm invalid.ignored
>
>.L20     MNOTE 4,'Length of block does not exceed 256 (&LBL DSECT), &&CX12345678
#^^^^ source.hlasm keyword.control.hlasm
#    ^^^^^^^^^^ source.hlasm entity.name.function.hlasm
#              ^ source.hlasm
#               ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                ^ source.hlasm
#                 ^ source.hlasm string.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                                       ^^^^ source.hlasm variable.other.readwrite.hlasm
#                                                           ^^^^^^^^^ source.hlasm string.hlasm
#                                                                    ^^ source.hlasm string.hlasm
#                                                                      ^ source.hlasm string.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               LEANUP option can be optimized to YES',OP2OP2OP2OP2OP2OA-12345678
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                                     ^ source.hlasm
#                                                      ^^^^^^^^^^^^^^^^^ source.hlasm punctuation.definition.tag.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARK,REMARK
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^ source.hlasm punctuation.definition.tag.hlasm
#                     ^ source.hlasm
#                      ^^^^^^ source.hlasm punctuation.definition.tag.hlasm
>
>assa
#^^^^ source.hlasm support.class.hlasm
>
>LABEL INSTR OP1,OP2,   REMARKS                                         -12345678
#^^^^^ source.hlasm support.class.hlasm
#     ^^^^^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^^^ source.hlasm punctuation.definition.tag.hlasm
#               ^ source.hlasm
#                ^^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^ source.hlasm
#                    ^ source.hlasm
#                     ^^ source.hlasm
#                       ^^^^^^^ source.hlasm comment.hlasm
#                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP3,OP4,       1                                        -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^ source.hlasm punctuation.definition.tag.hlasm
#                      ^ source.hlasm
#                       ^ source.hlasm
#                        ^^^^^^ source.hlasm
#                              ^ source.hlasm comment.hlasm
#                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP5,OP6                                                 -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^ source.hlasm punctuation.definition.tag.hlasm
#                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP6            1                                        -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm comment.hlasm
#                  ^^^^^^^^^^^^ source.hlasm
#                              ^ source.hlasm comment.hlasm
#                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP7                                                      12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm comment.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm invalid.ignored
>               
>.L20     MNOTE 4,'Length of block does not exceed 256 (&LBL DSECT), &&CX12345678
#^^^^ source.hlasm keyword.control.hlasm
#    ^^^^^^^^^^ source.hlasm entity.name.function.hlasm
#              ^ source.hlasm
#               ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                ^ source.hlasm
#                 ^ source.hlasm string.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                                       ^^^^ source.hlasm variable.other.readwrite.hlasm
#                                                           ^^^^^^^^^ source.hlasm string.hlasm
#                                                                    ^^ source.hlasm string.hlasm
#                                                                      ^ source.hlasm string.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               LEANUP option can be optimized to YES ,OP2               12345678
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                                                       ^^^^^^^^^ source.hlasm invalid.ignored
>
>LABEL INSTR OP1,OP2    REMARKS                                         -12345678
#^^^^^ source.hlasm support.class.hlasm
#     ^^^^^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^^^ source.hlasm punctuation.definition.tag.hlasm
#               ^ source.hlasm
#                ^^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^^^^ source.hlasm
#                       ^^^^^^^ source.hlasm comment.hlasm
#                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP3 OP4,       1                                        -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm comment.hlasm
#                  ^ source.hlasm
#                   ^^^^ source.hlasm comment.hlasm
#                       ^^^^^^^ source.hlasm
#                              ^ source.hlasm comment.hlasm
#                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP5,OP6                                                 -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^ source.hlasm comment.hlasm
#                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP6            1                                        -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm comment.hlasm
#                  ^^^^^^^^^^^^ source.hlasm
#                              ^ source.hlasm comment.hlasm
#                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP7                                                      12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm comment.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm invalid.ignored
>               